src/main/java/com/microsoft/azure/relay/CompletableFutureUtil.java

```diff
@@ -81,4 +81,8 @@ public T call() throws Exception {
 		}
 		return taskCF;
 	}
+	
+	static boolean isDoneNormally(CompletableFuture<?> cf) {
```

Comment:
>isDoneNormally [](start = 16, length = 14)

Since this is for callers outside this class I'd put it higher in the file than any private methods.